tools/cldr-code/src/main/java/org/unicode/cldr/util/LocalePathValueListMatcher.java

```diff
@@ -70,20 +82,33 @@ public static LocalePathValueListMatcher load(Stream<String> lines) {
 
     public boolean lookingAt(String locale, String path, String value) {
         for (LocalePathValueListMatcher.LocalePathValueMatcher lpv : matchData) {
-            if (!lpv.lookingAt(locale, path, value)) {
-                return false;
+            if (lpv.lookingAt(locale, path, value)) {
+                return true;
             }
         }
-        return true;
+        return false;
+    }
+
+    /** Just test the locale, to see if we need to bother */
+    public boolean lookingAt(String locale) {
+        for (LocalePathValueListMatcher.LocalePathValueMatcher lpv : matchData) {
+            if (lpv.lookingAt(locale)) {
+                return true;
+            }
+        }
+        return false;
     }
 
     private static void load(
             String line, List<LocalePathValueListMatcher.LocalePathValueMatcher> _matchData) {
         line = line.trim();
-        if (line.startsWith("#")) {
+        if (line.startsWith("#") || line.isEmpty()) {
```

Comment:
this explains those "null\tnull\tnull" I was seeing!

Reply:
Yup! When I first tested I had no blank lines!